### ISSUE TITLE
[Suggestion] Ability to update our own Steam Blueprints

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#2695]
Please correct me if I'm wrong and the option's already there and I cannot seem to find it, but I would like to be able to update my existing blueprints that I had published on the Steam Workshop BUT without using the original blueprint. Sadly, I no longer have it and it would be a huge convenience if I could update the thumbnail or blueprint itself.

**Comment by Womble-TC** [comment]
Yeah, this would be nice, really want to add piston landing gear to my X-wing. Would also be useful if/when reactors and stuff get replaced in the future by different types of power generation.

**Comment by tsunamayo** [comment]
Okay, it make sense indeed.
